### PR TITLE
fix: enable authzs MsgExec to be used in a blobTx

### PR DESCRIPTION
**Pull request by cmwaters**
Closes: https://github.com/celestiaorg/celestia-app/issues/1773

### CODE REVIEW

File: app/test/process_proposal_test.go [hunk 1]
```diff
@@ -299,10 +312,26 @@ func TestProcessProposal(t *testing.T) {
...
+			},
+			expectedResult: abci.ResponseProcessProposal_ACCEPT,
+		},
 	}
 
-	for _, tt := range tests {
+	for idx, tt := range tests {
+		if idx != 15 {
+			continue
+		}
```
Comment (rootulp) [inline app/test/process_proposal_test.go:330]: [blocking] I think this is residual from development

```suggestion
```

File: x/blob/types/blob_tx_test.go [hunk 2]
```diff
@@ -117,3 +124,67 @@ func randMsgPayForBlobsWithNamespaceAndSigner(t *testing.T, signer string, ns ap
...
+			msgs:   []sdk.Msg{&invalidMsgExec},
+			blobs:  []*tmproto.Blob{blob},
+			expErr: true,
+		},
+		{
+			name:   "blob tx using invalid msg exec that has multiple messages",
+			msgs:   []sdk.Msg{&msgExecWithTwoMsgs},
+			blobs:  []*tmproto.Blob{blob},
+			expErr: true,
+		},
```
Comment (rootulp) [inline x/blob/types/blob_tx_test.go:168]: 👍

**Review by rootulp — CHANGES_REQUESTED** [review]
LGTM after one comment

**Review by rootulp — APPROVED** [review]

**Comment by evan-forbes** [comment]
per synchronous discussions, I don't think we're moving forward with including authz blob support in the v1.0.0 release candidate, so should we still merge this?

**Comment by cmwaters** [comment]
> per synchronous discussions, I don't think we're moving forward with including authz blob support in the v1.0.0 release candidate, so should we still merge this?

Nope, If you don't mind I'm going to put this back into draft mode and we can reassess it after cutting v1.0.0

**Comment by cmwaters** [comment]
Closing as stale

**Pull request closed without merging**